### PR TITLE
feat cards

### PR DESCRIPTION
**Pull request by abhij1607**
image
image

- basic card
- card with badge
- card with text on image
- card with text on header
- card with only text
- card with horizontal image and text

### CODE REVIEW

File: components/cards/cards.css [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+.img-info-container {
+  position: relative;
+}
+.text-over-img {
+  position: absolute;
+  top: 0;
+}
+.text-over-img .card-title,
+.text-over-img .card-author {
+  color: white;
```
Comment (lannisterrr) [inline components/cards/cards.css:68]: using hex code / variables for colors ideally

**Review by lannisterrr — COMMENTED** [review]
Other than point mentioned above , code seems fine.

**Review by junaidshaikh-js — APPROVED** [review]
Everything looks fine. Clean code. All possible best practices is followed, can be merged.

File: components/cards/cards.css (hunk 1 above)
Comment (abhij1607) [reply components/cards/cards.css:68]: ok, will change them to variables later

**Pull request merged**